docs/design/coreclr/botr/porting-ryujit.md

```diff
@@ -1,37 +1,97 @@
 # RyuJIT: Porting to different platforms
 
-## What is a Platform?
-* Target instruction set and pointer size
-* Target calling convention
+First, understand the JIT architecture by reading [RyuJIT overview](ryujit-overview.md).
+
+# What is a Platform?
+
+* Target instruction set
+* Target pointer size
+* Target operating system
+* Target calling convention and ABI (Application Binary Interface)
 * Runtime data structures (not really covered here)
 * GC encoding
-  * So far only JIT32_GCENCODER and everything else
-* Debug info (so far mostly the same for all targets?)
-* EH info (not really covered here)
+  * So far only JIT32_GCENCODER (used by Windows x86) and everything else
+* Debug information (so far mostly the same for all targets?)
```

Comment:
Nit: what's the meaning of the question mark?

Reply:
The question mark means, I believe, the statement was made without understanding if it was entirely true. I'll just remove it (and simplify the statement).